spec/features/notifications_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe 'Activity notification alerts', js: true do
+  let(:agent) do
+    create(:agent)
+  end
+
+  let(:guider) do
+    create(:guider)
+  end
+
+  scenario 'A new high priority activity occurs' do
+    given_a_browser_session_for guider, agent do
```

Comment:
Can we parenthesise these user arguments everywhere we call this method? The way these steps are written makes it quite tricky to scan and pick them up.